doc/whats_new.rst

```diff
@@ -4639,236 +4637,75 @@ David Huard, Dave Morrill, Ed Schofield, Travis Oliphant, Pearu Peterson.
-
 .. _Brian McFee: https://bmcfee.github.io
 
-.. _Vighnesh Birodkar: https://github.com/vighneshbirodkar
-
-.. _Chyi-Kwei Yau: https://github.com/chyikwei
-.. _Martino Sorbaro: https://github.com/martinosorb
-.. _Jaidev Deshpande: https://github.com/jaidevd
-.. _Arthur Mensch: https://github.com/arthurmensch
-.. _Daniel Galvez: https://github.com/galv
-.. _Jacob Schreiber: https://github.com/jmschrei
-.. _Ankur Ankan: https://github.com/ankurankan
 .. _Valentin Stolbunov: http://www.vstolbunov.com
-.. _Jean Kossaifi: https://github.com/JeanKossaifi
-.. _Andrew Lamb: https://github.com/andylamb
-.. _Graham Clenaghan: https://github.com/gclenaghan
-.. _Giorgio Patrini: https://github.com/giorgiop
-.. _Elvis Dohmatob: https://github.com/dohmatob
-.. _yelite: https://github.com/yelite
-.. _Issam H. Laradji: https://github.com/IssamLaradji
-
-.. _Asish Panda: https://github.com/kaichogami
-
-.. _Philipp Dowling: https://github.com/phdowling
-
-.. _Imaculate: https://github.com/Imaculate
-
-.. _Bernardo Stein: https://github.com/DanielSidhion
-
-.. _Andrea Bravi: https://github.com/AndreaBravi
-
-.. _Devashish Deshpande: https://github.com/dsquareindia
-
-.. _Jonathan Arfa: https://github.com/jarfa
-
-.. _Anish Shah: https://github.com/AnishShah
-
-.. _Ryad Zenine: https://github.com/ryadzenine
-
-.. _Guillaume Lemaitre: https://github.com/glemaitre
-
-.. _JPFrancoia: https://github.com/JPFrancoia
-
-.. _Thierry Guillemot: https://github.com/tguillemot
-
-.. _Wei Xue: https://github.com/xuewei4d
 
-.. _Ori Ziv: https://github.com/zivori
+.. _Jaques Grobler: https://github.com/jaquesgrobler
 
-.. _Sears Merritt: https://github.com/merritts
-
-.. _Wenhua Yang: https://github.com/geekoala
-
-.. _Arnaud Fouchet: https://github.com/afouchet
-
-.. _Sebastian Säger: https://github.com/ssaeger
-
-.. _YenChen Lin: https://github.com/yenchenlin
-
-.. _Nelson Liu: https://github.com/nelson-liu
-
-.. _Manvendra Singh: https://github.com/manu-chroma
-
-.. _Ibraim Ganiev: https://github.com/olologin
-
-.. _Konstantin Podshumok: https://github.com/podshumok
-
-.. _David Staub: https://github.com/staubda
-
-.. _Hong Guangguo: https://github.com/hongguangguo
-
-.. _Mads Jensen: https://github.com/indianajensen
-
-.. _Sebastián Vanrell: https://github.com/srvanrell
-
-.. _Robert McGibbon: https://github.com/rmcgibbo
-
-.. _Gregory Stupp: https://github.com/stuppie
-
-.. _Russell Smith: https://github.com/rsmith54
+.. _Lars Buitinck: https://github.com/larsmans
 
-.. _Utkarsh Upadhyay: https://github.com/musically-ut
+.. _Loic Esteve: https://github.com/lesteve
 
-.. _Eugene Chen: https://github.com/eyc88
+.. _Noel Dawe: https://github.com/ndawe
 
-.. _Narine Kokhlikyan: https://github.com/NarineK
+.. _Raghav R V: https://github.com/raghavrv
 
-.. _Peng Meng: https://github.com/mpjlu
+.. _Tom Dupre la Tour: https://github.com/TomDLT
 
-.. _Dylan Werner-Meier: https://github.com/unautre
+.. _Nelle Varoquaux: https://github.com/nellev
 
```

Comment:
While we're at it, could you please remove all the blank lines in this target URL listing?